config/config.go

```diff
@@ -77,3 +84,11 @@ type Config struct {
 	TUI      TUIConfig  `mapstructure:"tui" json:"tui,omitempty" yaml:"tui"`
 	Cron     CronConfig `mapstructure:"cron" yaml:"cron" json:"cron,omitempty"`
 }
+
+func (cfg *Config) GetDatabase() (*bun.DB, error) {
+	if cfg.HTTP.Database.Driver == "sqllite" {
+		return sqlite.New(cfg.HTTP.Database.DSN, cfg.HTTP.Database.LogQueries)
+	}
+
+	return postgres.New(cfg.HTTP.Database.DSN, cfg.HTTP.Database.LogQueries)
+}
```

Comment:
also can we encapsulate this into just one `New` function and depending on the driver, we return the correctly configured one. having Postgres and sqlite packages just to export `New` seems overkill to me